with-graphql/stepzen.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "endpoint": "api/alternating-jaguar"
```

Comment:
Could you rename this  to `api/with-graphql`

Reply:
actually, just going to remove it -- otherwise I think users could run into issues with StepZen walking back up the tree and getting 'confused' between repos